src/components/shop/shop-titles.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @module components/shop
+ * @copyright BBC 2020
+ * @author BBC Children's D+E
+ * @license Apache-2.0
+ */
+
+import { createTitles } from "../select/titles.js";
+import { getSafeArea, getYPos, getScaleFactor } from "./shop-layout.js";
+
+export const createTitle = (scene, metrics) => {
+    const titleContainer = scene.add.container();
+    titleContainer.add(createTitles(scene));
+
+    const safeArea = getSafeArea(scene.layout);
```

Comment:
maybe the area gets given to this function, rather than it grabbing it + creating the title

Reply:
3d2ba13